feat(content): Free Worlds prologue: Pact Supplies mission chain (#11294)
endless-sky/endless-sky: data/human/human missions.txt

	`You enter the quartermaster's office, where you find a small group of captains already waiting, all of whom were tipped off about a job opportunity to support the militias in a supply mission. Before long, the quartermaster himself arrives alongside a militia officer. There is a respectful pause as the quartermaster goes to sit at his desk. As he is settled, he scans all of you down for a moment with a focused eye, and clears his throat.`
	`	"Good day, Captains. You may want to know why you are standing here in my office, and why my colleagues have asked you to help us on a 'supply job.' There is a developing situation in the Southern Rim that concerns our larger militia brothers, and we feel obliged to support them. Piracy is ever more prevalent down there, and we generally have a surplus of weaponry that we can manufacture and replace ourselves. Therefore, we are looking to move some of our weapons stores down to <planet> to help deal with their situation. We will gladly pay each of you <payment> to convoy these goods. Do any of you have questions?"`
	choice
		`	"Who are these weapons going to?"`
			goto pact
		`	"What's this 'developing situation'?"`

> 	`	"The pirates have been becoming more and more aggressive in the South. They've started doing more than just interdict merchants; they've been raiding planets for ships and tribute on an organized scale once again, and the Pact has requested more weaponry to deal with them. Let me tell you, though: we wouldn't have this problem if the Navy would stop hiding in the Paradise Worlds and start patrolling these regions again."`

Reviewer: ```suggestion
	`	"The pirates have been becoming more and more aggressive in the South. They've started doing more than just interdict merchants; they've been raiding planets for ships and tribute on an organized scale once again, and the Pact has requested more weaponry to deal with them. Let me tell you though - we wouldn't have this problem if the Navy would stop hiding in the Paradise Worlds and start patrolling these regions again."`
```

	label decision
	`	There's a shuffle as a few captains raucously walk out of the quartermaster's office.`
	`	As he leaves, one captain shouts, "We won't support any of you begging southerners here!" while another adds, "This is all going to some rebellion, and you know it! These Southern Pact goons are gun-runners!" Going by their accents and faces, those captains came from the Paradise Worlds, and they didn't take the quartermaster's comment so kindly.`
	`	While those captains leave, both the militia officer and the quartermaster hold their faces grim and firm. "Any of you are free to join them if you wish. We will not penalize you, but do know that you may miss out on an opportunity to help our Southern brothers. That may mean a lot in the future."`
	choice
		`	(Stay and accept the job.)`
			goto end
		`	(Leave with the Paradise Worlds captains.)`
			decline

	label end

> 	`	Your now smaller group of captains are led by the militia officer to one of the warehouses on the edge of the spaceport, where a store of energy blasters, beam lasers, and various pieces of ammunition are held. Stevedores rapidly empty the warehouse to fill your ships with cargo. As they work, the militia officer addresses all of you. "Our brothers down in the South on <planet> expect this delivery by <date>. We thank you for your assistance, and pray that you have a safe journey. Farewell."`

Reviewer: ```suggestion
	`	You and the now-smaller group of captains are led by the militia officer to one of the warehouses on the edge of the spaceport, where a store of energy blasters, beam lasers, and various pieces of ammunition are held. Stevedores rapidly empty the warehouse, filling the diverse range of ships with cargo. As they work, the militia officer addresses all of you: "Our brothers down in the South on <planet> expect this delivery by <date>. We thank you for your assistance, and pray that you have a safe journey. Farewell."`
```

	on offer
		event "stone waiting most" 14
		fail

fleet "Pact Supplies"
	government "Merchant"
	names "civilian"
	outfitters "Common Outfits" "Ammo South"
	personality
		confusion 30
	variant 1
		"Shuttle"
		"Star Barge" 2
		"Argosy"
		"Sparrow" 2

> 		"Hawk"

Reviewer 2: Why does this fleet have so many ships? Shouldn't they be split up?
Also might make sense to change the government from Merchant to Militia.

substitutions

> 	"<Pact / Free Worlds>" "South"

Reviewer 2: When is this substitution used?

Author: In case there is any event that the other two substitutions are no longer valid. The edge case is very slim but I thought might as well add it, it's all of one line.

Reviewer 2: Might be worth moving it below the other two, just so that it's easier to read, but sure.

Reviewer 3: Moving this below the other two substitutions would cause it to always override them, displaying "South".

For this line specifically, I'd much rather have `"<Pact / Free Worlds>" "Pact"` or `"<Pact / Free Worlds>" "Free Worlds"` be the default case than having both "Pact" and "Free Worlds" be conditional. There's no way for both `not "event: war begins"` and `has "event: war begins` to be true at the same time, so having a third, impossible case would just be confusing.

Author: ```suggestion
```

	"<Pact / Free Worlds>" "Pact"
		not "event: war begins"
	"<Pact / Free Worlds>" "Free Worlds"
		has "event: war begins"

mission "Pact Supplies Trigger"
	invisible
	landing
	minor
	to complete
		not "event: war begins"
		not "event: chosen sides"
		random < 40
	source
		attributes "dirt belt"
		attributes "factory"
		not planet "New Sahara"
	on offer
		conversation
			`<origin> is one of the few planets in the Dirt Belt with a developed industrial base. In addition to the horse-drawn carts of cereals, crops, and meat, there are also trucks and vans carrying locally manufactured goods from the cities nearby.`

> 			`	To protect this industry from the ever-increasing piracy threat, there is a substantial militia base on this planet that, in place of the Navy's retreating patrols, have taken up the position of the police forces in this area. Even the planetary spaceport is now staffed by the militia, and instead of the Gunboats that would have scanned you upon approach, there are Argosies and Bastions that scan you instead.`

Reviewer 2: Except for the militia don't actually scan you...
Maybe change it a bit, like this:
```suggestion
			`	To protect this industry from the ever-increasing piracy threat, there is a substantial militia base on this planet that, in place of the Navy's retreating patrols, have taken up the position of the police forces in this area. Even the planetary spaceport is now staffed by the militia, and instead of the Gunboats that would have scanned you upon approach, there are Argosies and Bastions that keep an eye on you instead.`
```

Author: Someone has to scan your goods when you land on a planet.

			`	As you sign off your landing documents with the militia officer on duty, she asks you to look out on local job boards "if you are willing to help the militias on a supply job." She then thanks you for your cooperation and swiftly moves on to the next ship.`
				decline

mission "Pact Supplies Convoy"
	job
	name "Transport weapons to <planet>"
	description "The local militia is requesting merchant captains to participate in a convoy supplying <cargo> to <destination>. Available captains are to report to the quartermaster's office. Payment is <payment>."
	cargo "weapons" 20
	destination "Longjump"
	source
		attributes "dirt belt"
		attributes "factory"
		not planet "New Sahara"
	to offer
		or
			has "ships: Transport"
			has "ships: Light Freighter"
			has "ships: Heavy Freighter"
		not "Pact Supplies Escort: completed"
	on accept
		conversation "Pact Supplies"
	npc accompany save
		government "Merchant"
		personality escort timid

> 		fleet "Pact Supplies"

Reviewer 2: Ah, now I see... wouldn't it be cleaner to define the fleet here?

Author: Saves a little bit of space defining it only once, whatever you think is best.

	npc
		government "Pirate"
		personality plunders waiting
		fleet "Small Southern Pirates" 2
	on visit
		dialog `You have reached <planet>, but you're missing something! Either you left part of the convoy behind, or not all of the cargo is in the system.`
	on complete
		"assisted the free worlds" ++
		payment 35000 50

> 		dialog `After you land at <planet>'s spaceport, a militia officer comes to inspect your ship before giving it the all-clear to be unloaded. "Thank you for your service. I will make sure that the <Pact / Free Worlds> will remember your assistance." He hands you your payment of <payment> and makes his way to the next convoy ship.`

Reviewer 2: ```suggestion
		dialog `After you land at <planet>'s spaceport, a militia officer comes to inspect your ship before giving it the all-clear to be unloaded. "Thank you for your service. I will make sure that the <Pact / Free Worlds> remembers your assistance." He hands you your payment of <payment> and makes his way to the next convoy ship.`
```

mission "Pact Supplies Escort"
	job
	name "Escort weapons convoy to <planet>"
	description "The local militia is requesting merchant captains to escort a convoy supplying weapons to <destination>. Available captains are to report to the quartermaster's office. Payment is <payment>."
	destination "Longjump"
	source
		attributes "dirt belt"
		attributes "factory"
		not planet "New Sahara"
	to offer
		or
			has "ships: Interceptor"
			has "ships: Light Warship"
			has "ships: Medium Warship"
			has "ships: Heavy Warship"
		not "Pact Supplies Convoy: completed"
	on accept
		conversation "Pact Supplies"

	npc accompany save
		government "Merchant"
		personality escort timid
		fleet "Pact Supplies"
	npc
		government "Pirate"
		personality plunders waiting
		fleet "Small Southern Pirates" 2
	on visit
		dialog `You have reached <planet>, but you left part of the convoy behind! Better depart and wait for them to arrive in this star system.`
	on complete
		"assisted the free worlds" ++
		payment 35000 50

> 		dialog `After you land at <planet>'s spaceport, a militia officer comes to greet you. "Thank you for your service. I will make sure that the <Pact / Free Worlds> will remember your assistance." He hands you your payment of <payment>, and makes his way to the next convoy ship.`

Reviewer 2: ```suggestion
		dialog `After you land at <planet>'s spaceport, a militia officer comes to greet you. "Thank you for your service. I will make sure that the <Pact / Free Worlds> remembers your assistance." He hands you your payment of <payment>, and makes his way to the next convoy ship.`
```

conversation "Pact Supplies"
	branch trigger
		has "Pact Supplies Trigger: offered"
	`You make your way to the quartermaster's office, where you find a small group of captains already waiting. Many of them are talking about how they were tipped off about a job opportunity to support the militias in a supply mission. Before long, the quartermaster arrives alongside a militia officer.`
		goto main

	label trigger
	`You make your way to the quartermaster's office, where you find a small group of captains already waiting. Like you, many of them were tipped off about a job opportunity to support the militias in a supply mission. Before long, the quartermaster arrives alongside a militia officer.`

	label main

> 	`	There is a respectful pause as the quartermaster goes to sit at his desk. As he is settled, he scans all of you down for a moment and clears his throat.`

Reviewer 2: ```suggestion
	`	There is a respectful pause as the quartermaster goes to sit at his desk. As he is settled, he scans all of you for a moment and clears his throat.`
```

	`	"Good day, Captains. You may want to know why you are standing here in my office, and why my colleagues have asked you to help us on a 'supply job.' There is a developing situation in the Southern Rim that concerns our larger militia brothers, and we feel obliged to support them. Piracy is ever more prevalent down there, and we generally have a surplus of weaponry that we can manufacture and replace ourselves. Therefore, we are looking to move some of our weapons stores down to <planet> to help them deal with their situation. We will gladly pay each of you <payment> to convoy these goods. Do any of you have questions?"`
	choice
		`	"Who are these weapons going to?"`
			goto pact

> 		`	"What's this 'developing situation?'"`

Reviewer 2: Maybe add a `(Remain silent.)` choice and someone else asks these questions?

	`	"The pirates have been becoming more and more aggressive in the South. They've started doing more than just interdict merchants; they've been raiding planets for ships and tribute on an organized scale once again, and the Pact has requested more weaponry to deal with them. Let me tell you though - we wouldn't have this problem if the Navy would stop hiding in the Paradise Worlds and start patrolling these regions again."`
		goto decision

	label pact

> 	`	"Many of the ships in our militia have affiliations with the Southern Mutual Defense Pact, which is a collection of planetary militia organizations. They're centered in the Rim and the South so we are not directly tied to them, but recently we have begun to cooperate with them as the Navy has been cowering out in the Paradise Worlds. Whatever is going on down south that affects the Pact will also affect us some way in another."`

Reviewer 2: ```suggestion
	`	"Many of the ships in our militia have affiliations with the Southern Mutual Defense Pact, which is a collection of planetary militia organizations. They're centered in the Rim and the South, so we're not directly tied to them, but recently we have begun to cooperate with them as the Navy has been cowering out in the Paradise Worlds. Whatever is going on down south that affects the Pact will also affect us some way in another."`
```

Author: I thought the quartermaster as a sorta formal guy, so I didn't use contractions all that much in his speech

Reviewer 2: ```suggestion
	`	"Many of the ships in our militia have affiliations with the Southern Mutual Defense Pact, which is a collection of planetary militia organizations. They're centered in the Rim and the South, so we are not directly tied to them, but recently we have begun to cooperate with them as the Navy has been cowering out in the Paradise Worlds. Whatever is going on down south that affects the Pact will also affect us some way in another."`
```

	label decision

> 	`	There's a shuffle as a few captains raucously walk out of the quartermaster's office. As he leaves one captain shouts, "You begging southerners should be grateful for the Navy's protection!" while another adds, "This is all going to some rebellion, and you know it! These Southern Pact goons are gun-runners!" Going by their accents and faces, those captains came from the Paradise Worlds, and they didn't take the quartermaster's comment about the Navy so kindly.`

Reviewer 2: ```suggestion
	`	There's a shuffle as a few captains raucously walk out of the quartermaster's office. As he leaves one captain shouts, "You begging southerners should be grateful for the Navy's protection!" while another adds, "This is all going to some rebellion, and you know it! These Southern Pact goons are gun-runners!" Going by their accents and faces, those captains came from the Paradise Worlds, and they didn't take the quartermaster's comment about the Navy kindly.`
```

Author: I think it's fine to keep it as `so kindly`, personally

Reviewer 2: I suppose it could go either way. If you like this better, that's fine.

	on offer
		event "stone waiting most" 14
		fail

fleet "Pact Supplies"
	government "Merchant"
	names "civilian"
	outfitters "Common Outfits" "Ammo South"
	personality
		confusion 30
	variant 1
		"Shuttle"
		"Star Barge" 2
		"Argosy"
		"Sparrow" 2
		"Hawk"

substitutions
	"<Pact / Free Worlds>" "South"
	"<Pact / Free Worlds>" "Pact"
		not "event: war begins"
	"<Pact / Free Worlds>" "Free Worlds"
		has "event: war begins"

mission "Pact Supplies Trigger"
	invisible
	landing
	minor

> 	to complete

Reviewer 2: ```suggestion
	to offer
```

		not "event: war begins"
		not "event: chosen sides"
		random < 40
	source
		attributes "dirt belt"
		attributes "factory"
		not planet "New Sahara"
	on offer
		conversation
			`<origin> is one of the few planets in the Dirt Belt with a developed industrial base. In addition to the horse-drawn carts of cereals, crops, and meat, there are also trucks and vans carrying locally manufactured goods from the cities nearby.`

> 			`	To protect this industry from the ever-increasing piracy threat, there is a substantial militia base on this planet that, in place of the Navy's retreating patrols, have taken up the position of the police forces in this area. Even the planetary spaceport is now staffed by the militia, and instead of the Gunboats that would have scanned you upon approach, there are Argosies and Bastions that scan you instead.`

Reviewer 2: ```suggestion
			`	To protect this industry from the ever-increasing piracy threat, there is a substantial militia base on this planet that, in place of the Navy's retreating patrols, has taken up the position of the police forces in this area. Even the planetary spaceport is now staffed by the militia, and instead of the Gunboats that would have scanned you upon approach, there are Argosies and Bastions that perform the same duties.`
```
Removing the use of "scan" twice, and "a" militia base wouldn't be plural.

Author: ```suggestion
			`	To protect this industry from the ever-increasing piracy threat, there is a substantial militia presence on this planet that, in light of the Navy's retreating patrols, has taken up the position of the police forces in this area. Even the planetary spaceport is now staffed by the militia, and instead of the Gunboats that would have scanned you upon approach, there are Argosies and Bastions in their place.`
```

> 			`	As you sign off your landing documents with the militia officer on duty, she asks you to look out on local job boards "if you are willing to help the militias on a supply mission." She then thanks you for your cooperation and swiftly moves on to the next ship.`

Reviewer 4: "look out on" is a bit awkward. Maybe this instead?

```suggestion
			`	As you sign off your landing documents with the militia officer on duty, she asks you to check local job boards "if you are willing to help the militias on a supply mission." She then thanks you for your cooperation and swiftly moves on to the next ship.`
```

				decline

mission "Pact Supplies Convoy"
	job
	name "Transport weapons to <planet>"
	description "The local militia is requesting merchant captains to participate in a convoy supplying <cargo> to <destination>. Available captains are to report to the quartermaster's office. Payment is <payment>."
	cargo "weapons" 20
	destination "Longjump"
	source
		attributes "dirt belt"
		attributes "factory"

> 		not planet "New Sahara"

Reviewer 2: Why not?

Author: New Sahara is so far away from the Southern Pact planets that it would be unlikely for them to send anything over that way. In addition it's not very inhabited and probably shares more in common with the Near Earth/Cradle planets.

	to offer
		or
			has "ships: Transport"
			has "ships: Light Freighter"
			has "ships: Heavy Freighter"

> 			has "ships: Utility"

Reviewer 2: ```suggestion
			has "ships: Utility"
			has "ships: Space Liner"
```

> 		not "Pact Supplies Escort: completed"

Reviewer 2: ```suggestion
		not "Pact Supplies Escort: done"
```

	on accept
		conversation "Pact Supplies"
	npc accompany save
		government "Merchant"
		personality escort timid
		fleet "Pact Supplies"
	npc
		government "Pirate"
		personality plunders waiting

> 		fleet "Small Southern Pirates" 2

Reviewer 2: How about pirates at the destination as well?

	on visit

> 		dialog `You have reached <planet>, but you're missing something! Either you left part of the convoy behind, or not all of the cargo is in the system.`

Reviewer 4: This could almost be a dialog phrase...

	on complete

> 		"assisted the free worlds" ++

Reviewer 2: ```suggestion
		"assisted free worlds" ++
```

		payment 35000 50
		dialog `After you land at <planet>'s spaceport, a militia officer comes to inspect your ship before giving it the all-clear to be unloaded. "Thank you for your service. I will make sure that the <Pact / Free Worlds> remembers your assistance." He hands you your payment of <payment> and makes his way to the next convoy ship.`

mission "Pact Supplies Escort"
	job
	name "Escort weapons convoy to <planet>"
	description "The local militia is requesting merchant captains to escort a convoy supplying weapons to <destination>. Available captains are to report to the quartermaster's office. Payment is <payment>."
	destination "Longjump"
	source
		attributes "dirt belt"
		attributes "factory"
		not planet "New Sahara"
	to offer
		or
			has "ships: Interceptor"
			has "ships: Light Warship"
			has "ships: Medium Warship"
			has "ships: Heavy Warship"
			has "ships: Superheavy"

> 		not "Pact Supplies Convoy: completed"

Reviewer 2: ```suggestion
		not "Pact Supplies Convoy: done"
```

	on accept
		conversation "Pact Supplies"

	npc accompany save
		government "Merchant"
		personality escort timid
		fleet "Pact Supplies"
	npc
		government "Pirate"
		personality plunders waiting
		fleet "Small Southern Pirates" 2
	on visit
		dialog `You have reached <planet>, but you left part of the convoy behind! Better depart and wait for them to arrive in this star system.`
	on complete

> 		"assisted the free worlds" ++

Reviewer 2: ```suggestion
		"assisted free worlds" ++
```

		payment 35000 50

> 		dialog `After you land at <planet>'s spaceport, a militia officer comes to greet you. "Thank you for your service. I will make sure that the <Pact / Free Worlds> remembers your assistance." He hands you your payment of <payment>, and makes his way to the next convoy ship.`

Reviewer 4: I notice that all your generic militia officers are male. Maybe mix it up a bit or use they/their as pronouns instead?

Author: No, the militia officer who informs you of the job offer is female.

Reviewer 4: Ah, I obviously can't read.

conversation "Pact Supplies"
	branch trigger
		has "Pact Supplies Trigger: offered"
	`You make your way to the quartermaster's office, where you find a small group of captains already waiting. Many of them are talking about how they were tipped off about a job opportunity to support the militias in a supply mission. Before long, the quartermaster arrives alongside a militia officer.`
		goto main

	label trigger
	`You make your way to the quartermaster's office, where you find a small group of captains already waiting. Like you, many of them were tipped off about a job opportunity to support the militias in a supply mission. Before long, the quartermaster arrives alongside a militia officer.`

	label main

> 	`	There is a respectful pause as the quartermaster goes to sit at his desk. As he is settled, he scans all of you for a moment and clears his throat.`

Reviewer 2: ```suggestion
	`	There is a respectful pause as the quartermaster goes to sit at his desk. As he settles, he scans all of you for a moment and clears his throat.`
```

> 	`	"Good day, Captains. You may want to know why you are standing here in my office, and why my colleagues have asked you to help us on a 'supply job.' There is a developing situation in the Southern Rim that concerns our larger militia brothers, and we feel obliged to support them. Piracy is ever more prevalent down there, and we generally have a surplus of weaponry that we can manufacture and replace ourselves. Therefore, we are looking to move some of our weapons stores down to <planet> to help them deal with their situation. We will gladly pay each of you <payment> to convoy these goods. Do any of you have questions?"`

Reviewer 2: ```suggestion
	`	"Good day, captains. You may want to know why you are standing here in my office, and why my colleagues have asked you to help us on a 'supply job.' There is a developing situation in the Southern Rim that concerns our larger militia brothers, and we feel obliged to support them. Piracy is ever more prevalent down there, and we generally have a surplus of weaponry that we can manufacture and replace ourselves. Therefore, we are looking to move some of our weapons stores down to <planet> to help them deal with their situation. We will gladly pay each of you <payment> to convey these goods. Do any of you have questions?"`
```

	choice
		`	(Remain silent.)`
		`	"Who are these weapons going to?"`
			goto pact
		`	"What's this 'developing situation?'"`
			goto situation

> 	`	After a moment of silence, one of the other captains steps forward and asks sneeringly, "How do we know these weapons aren't going to be used by illegal actors, my good quartermaster? Weapons trafficking can be a serious crime in the Republic."`

Reviewer 2: ```suggestion
	`	After a moment, one of the other captains steps forward and asks sneeringly, "How do we know these weapons aren't going to be used by illegal actors, my good quartermaster? Weapons trafficking can be a serious crime in the Republic."`
```
Given that the player just said they'd be silent, we don't need to tell them it's silent.

> 	`	"You have no reason for suspicion," the quartermaster responds. "The weapons are being sent to our partners in the Southern Mutual Defense Pact, who have a much greater need for these munitions and outfits than we do, given the rates of piracy they're suffering through at the moment. We wouldn't need to do anything if it weren't for the Navy and their masters in Parliament sitting on their cushioned chairs every day and refusing to do anything about the situation while southron lives and ships are lost."`

Reviewer 2: ```suggestion
	`	"You have no reason for suspicion," the quartermaster responds. "The weapons are being sent to our partners in the Southern Mutual Defense Pact, who have a much greater need for these munitions and outfits than we do, given the rates of piracy they're suffering through at the moment. We wouldn't need to do anything if it weren't for the Navy and their masters in Parliament sitting on their cushioned chairs every day and refusing to do anything about the situation while Southern lives and ships are lost."`
```
~~I'm assuming this isn't a reference to Lord of the Rings~~

Author: I was more worried someone would see it as a Song of Ice and Fire reference

Also, southron has been used IRL, according to Wikipedia.

Author: On the other hand this will be the only mention of southron in the game. Do we want to set a precedent or do we want to conform? ~~I like setting precedents, as a law student.~~

Author: ```suggestion
	`	"You have no reason for suspicion," the quartermaster responds. "The weapons are being sent to our partners in the Southern Mutual Defense Pact, who have a much greater need for these munitions and outfits than we do, given the rates of piracy they're suffering through at the moment. We wouldn't need to do anything if it weren't for the Navy and their masters in Parliament sitting on their cushioned chairs every day and refusing to do anything about the situation while southern lives and ships are lost."`
```

> 	`	The air in the office seems to become more frigid as the captain responds, "I asked you whether these weapons wouldn't fall into the hands of privateers and inciters like the Pact, and instead all you do is insult the Navy and tell us we should go help the Pact because Parliament made you do so. You begging southerners should be grateful for the Navy's protection!" He turns around and pushes through the still crowd in the room. A few others turn and follow him, adding, "This is all going to some rebellion, and you know it! These Southern Pact goons are gun-runners!"`

Reviewer 4: ```suggestion
	`	The air in the office seems to become more frigid as the captain responds, "I asked you whether these weapons wouldn't fall into the hands of privateers and inciters like the Pact, and instead all you do is insult the Navy and tell us we should go help the Pact because Parliament made you do so. You begging Southerners should be grateful for the Navy's protection!" He turns around and pushes through the still crowd in the room. A few others turn and follow him, adding, "This is all going to some rebellion, and you know it! These Southern Pact goons are gun-runners!"`
```

Maybe not necessary, but should probably be standardized across these missions at least.

Reviewer 2: Well, I'd imagine that capitalizing it is a way of giving it distinction. Would a captain who says he doesn't like the South capitalize it?
Southern Pact is capitalized because it's a name, but southerners could almost be an adjective.

Author: As far as I've experienced, people don't capitalise "westerners" when they refer to people from Europe or North America.

	`	Going by their accents and faces, those captains came from the Paradise Worlds, and they didn't take the quartermaster's comment about the Navy or the Republic so kindly.`
		goto decision

	label situation

> 	`	"The pirates have been becoming more and more aggressive in the South. They've started doing more than just interdict merchants; they've been raiding planets for ships and tribute on an organized scale once again, and the Pact has requested more weaponry to deal with them. Let me tell you though - we wouldn't have this problem if the Navy would stop hiding in the Paradise Worlds and start patrolling these regions again."`

Reviewer 2: ```suggestion
	`	"The pirates have been becoming more and more aggressive in the South. They've started doing more than just interdict merchants; they've been raiding planets for ships and tribute on an organized scale once again, and the Pact has requested more weaponry to deal with them. Let me tell you, though - we wouldn't have this problem if the Navy would stop hiding in the Paradise Worlds and start patrolling these regions again."`
```
I _think_ the comma makes it sound better, but if you disagree feel free to resolve this.

		goto raucous

	label pact

> 	`	"Many of the ships in our militia have affiliations with the Southern Mutual Defense Pact, which is a collection of planetary militia organizations. They're centered in the Rim and the South, so we are not directly tied to them, but recently we have begun to cooperate with them as the Navy has been cowering out in the Paradise Worlds. Whatever is going on down south that affects the Pact will also affect us some way in another and so we are sending our surplus to them in support of their situation."`

Reviewer 2: ```suggestion
	`	"Many of the ships in our militia have affiliations with the Southern Mutual Defense Pact, which is a collection of planetary militia organizations. They're centered in the Rim and the South, so we are not directly tied to them, but recently we have begun to cooperate with them as the Navy has been cowering out in the Paradise Worlds. Whatever is going on down south that affects the Pact will also affect us some way or another, and so we are sending our surplus to them in support of their situation."`
```

> 	`	"Many of the ships in our militia have affiliations with the Southern Mutual Defense Pact, which is a collection of planetary militia organizations. They're centered in the Rim and the South, so we are not directly tied to them, but recently we have begun to cooperate with them as the Navy has been cowering out in the Paradise Worlds. Whatever is going on down south that affects the Pact will also affect us some way in another and so we are sending our surplus to them in support of their situation."`

Reviewer 4: Probably this would be best:

```suggestion
	`	"Many of the ships in our militia have affiliations with the Southern Mutual Defense Pact, which is a collection of planetary militia organizations. They're centered in the Rim and the South, so we are not directly tied to them, but recently we have begun to cooperate with them as the Navy has been cowering out in the Paradise Worlds. Whatever is going on down south that affects the Pact will also affect us in some way or another, so we are sending our surplus to them in support of their situation."`
```

	label raucous
	`	There's a shuffle as a few captains raucously walk out of the quartermaster's office. As he leaves one captain shouts, "You begging southerners should be grateful for the Navy's protection!" while another adds, "This is all going to some rebellion, and you know it! These Southern Pact goons are gun-runners!" Going by their accents and faces, those captains came from the Paradise Worlds, and they didn't take the quartermaster's comment about the Navy so kindly.`

	label decision

> 	`	As those captains leave, both the militia officer and the quartermaster hold their faces grim and firm. "Any of you are free to join them if you wish. We will not penalize you, but do know that you are missing out on an opportunity to help our Southern brothers. That may mean a lot in the future."`

Reviewer 2: "hold their faces" is a strange expression. Perhaps it could be changed; something like this?
```suggestion
	`	As those captains leave, both the militia officer and the quartermaster look grim and firm. "Any of you are free to join them if you wish. We will not penalize you, but do know that you are missing out on an opportunity to help our Southern brothers. That may mean a lot in the future."`
```

Author: Better?
```suggestion
	`	The militia officer and the quartermaster look on grimly as the captains leave. "Any of you are free to join them if you wish. We will not penalize you, but do know that you are missing out on an opportunity to help our Southern brothers. That may mean a lot in the future."`
```

	choice
		`	(Stay and accept the job.)`

> 			goto end
> 		`	(Leave with the Paradisian captains.)`
> 			decline
> 
> 	label end

Reviewer 2: ```suggestion
		`	(Leave with the Paradisian captains.)`
			decline

```

> 	`	You and the now-smaller group of captains are led by the militia officer to one of the warehouses on the edge of the spaceport, where a store of energy blasters, beam lasers, and various pieces of ammunition are held. Stevedores rapidly empty the warehouse, filling the diverse range of ships with cargo. As they work, the militia officer addresses all of you: "Our brothers down in the South on <planet> are counting on you. We thank you for your assistance, and pray that you have a safe journey. Farewell."`

Reviewer 2: At the moment, there's a chance the fleet will be carrying engines or rifles. Would it be worth specifying that they might do that here?
